angular/projects/spark-angular/src/lib/components/sprk-icon/sprk-icon.component.ts

```diff
@@ -41,16 +66,24 @@ export class SprkIconComponent {
    */
   public icon: string;
 
+  /**
+   * @ignore
+   */
+  getViewBox(): string {
```

Comment:
Does this need to be a function or can we set '0 0 64 64' as the default input?

Reply:
it was so weird, I tried that but it really didn't like it. I'll try it again just to make sure.
I went on to research how viewbox is weird when using [attr] 

https://docs.angularjs.org/guide/interpolation#-ngattr-for-binding-to-arbitrary-attributes

will report back

Reply:
lol jk seems like it was fine -- when I was trying it earlier I kept saying that viewBox wasn't defined. Maybe some typo

Reply:
✅ fixed this and pushed up